test/unit/getclient-spec.ts

```diff
@@ -2,124 +2,8 @@ import chai = require("chai");
 import chaiAsPromised = require("chai-as-promised");
 import "mocha";
 import "process";
-import { getClient } from "../../src";
-import { Constants, DurableOrchestrationClient } from "../../src/classes";
-import { TestConstants } from "../testobjects/testconstants";
-import { TestUtils } from "../testobjects/testutils";
 
-const expect = chai.expect;
 chai.use(chaiAsPromised);
 
-describe("getClient()", () => {
-    const defaultTaskHub = "TestTaskHub";
-    const defaultConnection = "Storage";
-
-    const defaultClientInputData = TestUtils.createOrchestrationClientInputData(
-        TestConstants.idPlaceholder,
-        Constants.DefaultLocalOrigin,
-        defaultTaskHub,
-        defaultConnection
-    );

```

Comment:
should we comment this out instead of deleting them? Might make it easier to recover once we re-enable them.

Reply:
Initially, I had it commented out, but instead removed it as per Eric's suggestion here: https://github.com/Azure/azure-functions-durable-js/pull/389#discussion_r989458594. I don't mind either way. Regardless, I also have an issue to make sure we don't forget about these tests: https://github.com/Azure/azure-functions-durable-js/issues/393